kong/plugins/cors/handler.lua

```diff
@@ -25,21 +25,26 @@ local function configure_origin(ngx, conf)
   if #conf.origins == 1 then
     if conf.origins[1] == "*" then
       ngx.ctx.cors_allow_all = true
+      ngx.header["Access-Control-Allow-Origin"] = "*"
+      return
 
     else
```

Comment:
because of the new `return` in the previous branch, this `else` becomes obsolete, better remove it